lib/presentation/auth/widgets/enter_username.dart

```diff
@@ -54,17 +54,19 @@ class _EnterUserNameState extends State<EnterUserName> {
             const SizedBox(height: 45.0),
             Form(
               key: _formKey,
-              onChanged: () => setState(() =>
-                  _isNameValid = _formKey.currentState?.validate() == true),
+              onChanged: () => setState(
+                () => _isNameValid = _formKey.currentState?.validate() == true,
+              ),
               child: TextFormField(
                 controller: _usernameController,
                 onChanged: (username) => _username = username,
                 style: const TextStyle(fontSize: 20.0),
                 keyboardType: TextInputType.text,
                 decoration: const InputDecoration(
-                    labelText: 'Your name',
-                    helperText: "Use your real name or choose a user name",
-                    focusColor: kAccentColor),
+                  labelText: 'Your name',
+                  helperText: "Use your real name or choose a user name",
```

Comment:
Maybe "or an alias" would be more accurate.

Reply:
I would stay with username since it's the most commonly used.